setup.py

```diff
@@ -31,7 +31,6 @@
 DEPENDENCIES = [
     'six',
```

Comment:
The six dependency can be replaced and removed.

Running `pyupgrade **/**.py --py3-plus` can help, and also upgrade a lot of other syntax.

https://github.com/asottile/pyupgrade